packages/protocol/contracts/thirdparty/LibSecureMerkleTrie.sol

```diff
@@ -67,3 +67,27 @@ library LibSecureMerkleTrie {
         return abi.encodePacked(keccak256(_key));
     }
 }
+
+// (The MIT License)
+
+// Copyright 2020-2021 Optimism
```

Comment:
Not all the thirdparty libs are from Optimsm though!

Reply:
i double checked, it's fine. i attributed optimism, zkevm circuit team, and openzeppelin teams for their respective contributions.